params/robot_localization_config.yaml

```diff
@@ -1,9 +1,15 @@
 # The frequency, in Hz, at which the filter will output a position estimate
 frequency: 30
 
+# Enable fusing of old data after newer data is received
+smooth_lagged_data: true
+
+# Length of time to keep data to refuse if older data is received
+history_length: 1000.0
```

Comment:
Shouldn't this be set to like 600ms since thats probably around the maximum we can reprocess without lagging?

Reply:
I believe some funky things were happening when this was shorter, which is why it's so long